pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -287,7 +287,44 @@ func describeProjectAndServer(project *projectapi.Project, server string) string
 	}
 }
 
-func describeDeploymentInServiceGroup(deploy graphview.DeploymentConfigPipeline) []string {
+func describeImageStreamTagRuntimeStatus(imageSpec string, c client.Interface, namespace string) string {
+	segments := strings.Split(imageSpec, "/")
+	streamPlusTag := ""
+	project := ""
+	if len(segments) == 2 {
+		project = segments[0]
+		streamPlusTag = segments[1]
+	} else {
+		streamPlusTag = imageSpec
+	}
+
+	segments = strings.SplitN(streamPlusTag, ":", 2)
+	if len(segments) == 2 {
+		stream := segments[0]
+		tag := segments[1]
+		is, err := c.ImageStreams(namespace).Get(stream)
+
+		if err != nil {
+			if project == namespace {
+				return fmt.Sprintf("the image stream %s in namespace %s is inaccessible with error %v", stream, namespace, err)
+			} else {
+				return ""
```

Comment:
should not eat the error.  why is this different from the project==namespace case?